static/metricsgraphics/css/metricsgraphics.css

```diff
@@ -0,0 +1,391 @@
+.mg-active-datapoint {
+    fill: black;
+    font-size: 0.9rem;
+    font-weight: 400;
+    opacity: 0.8;
+}
+
+.mg-area1-color {
+    fill: #0000ff;
+}
+
+.mg-area2-color {
+    fill: #05b378;
+}
+
+.mg-area3-color {
+    fill: #db4437;
+}
+
+.mg-area4-color {
+    fill: #f8b128;
+}
+
+.mg-area5-color {
+    fill: #5c5c5c;
+}
+
+text.mg-barplot-group-label {
+    font-weight:900;
+}
+
+.mg-barplot rect.mg-bar {
+    shape-rendering: auto;
+}
+
+.mg-barplot rect.mg-bar.default-bar {
+    fill: #b6b6fc;
+}
+
+.mg-barplot rect.mg-bar.default-active {
+    fill: #9e9efc;
+}
+
+.mg-barplot .mg-bar-prediction {
+    fill: #5b5b5b;
+}
+
+.mg-barplot .mg-bar-baseline {
+    stroke: #5b5b5b;
+    stroke-width: 2;
+}
+
+.mg-bar-target-element {
+    font-size:11px;
```

Comment:
These four CSS lines are not formatted to the same standard as the rest of this file. Perhaps a lint pass someday?

Reply:
before this lands I was going to auto-reformat everything. New policy for reviews is I don't even bother asking people to style things until the end.

Reply:
This comes from the external MetricsGraphics.js project library -- it's not my code.